src/electionguard/guardian.py

```diff
@@ -84,8 +84,23 @@ class Guardian(ElectionObjectBase):
     """
 
     def __init__(
-        self, id: str, sequence_order: int, number_of_guardians: int, quorum: int,
+        self,
+        id: str,
+        sequence_order: int,
+        number_of_guardians: int,
+        quorum: int,
+        nonce_seed: Optional[ElementModQ] = None,
     ) -> None:
+        """
+        Initialize a guardian with the specified arguments
+
+        :param id: the unique identifier for the guardian
+        :param sequence_order: a unique number in [0, 256) that identifies this guardian
+        :param number_of_guardians: the total number of guardians that will participate in the election
+        :param quorum: the count of guardians necessary to decrypt
+        :param nonce_seed: an optional `ElementModQ` value that can be used to generate the `ElectionKeyPair`.  
```

Comment:
josh has expressed some concerns that by allowing this to be provided, it may lead to users generating insecure secrets, but i think we should probably leave it for testing purposes, personally.